src/backend/Aggregator/Domains/Aggregation/AlbumList.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Domains.Aggregation
+{
+    public class AlbumList
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string CoverUrl { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this property type to 'System.Uri'.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5020178)